khtools/commandline.py

```diff
@@ -10,24 +10,27 @@
 import click
 
 # Within-module imports
-from khtools.hello import hello
```

Comment:
so hello is not used anywhere anymore? it is not added to the commandline here using click and its removed from the docs. can we remove the module too? just curious